src/theme/BlogPostItem/index.js

```diff
@@ -6,49 +6,52 @@
  */
 import React from 'react';
 import clsx from 'clsx';
-import { MDXProvider } from '@mdx-js/react';
-import Head from '@docusaurus/Head';
+import {MDXProvider} from '@mdx-js/react';
+import Translate from '@docusaurus/Translate';
 import Link from '@docusaurus/Link';
 import MDXComponents from '@theme/MDXComponents';
-import useBaseUrl from '@docusaurus/useBaseUrl';
+import Seo from '@theme/Seo';
 import styles from './styles.module.css';
-const MONTHS = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```

Comment:
FYI I "reswizzled" the component to update it to latest Docusaurus code (which is using prettier for better js code formatting now)

So most of the changes here are related to formatting and updated Docusaurus code